src/main/java/camp/nextstep/edu/step3/Hit.java

```diff
@@ -0,0 +1,46 @@
+package camp.nextstep.edu.step3;
+
+import java.util.Arrays;
+
+public enum Hit {
+    ZERO(0, 0),
+    ONE(1, 0),
+    TWO(2, 0),
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 1500000),
+    ALL(6, 2000000000);
+
+    private final int count;
+    private final int prizeMoney;
+
+    Hit(int count, int prizeMoney) {
+        this.count = count;
+        this.prizeMoney = prizeMoney;
+    }
+
+    static Hit valueOf(final int hitCount) {
+        return Hit.values()[hitCount];
+    }
+
+    public static Hit[] winningList() {
+        return Arrays.stream(Hit.values()).filter(Hit.TWO::isLow).toArray(Hit[]::new);
```

Comment:
```suggestion
        return Arrays.stream(Hit.values())
        .filter(Hit.TWO::isLow)
        .toArray(Hit[]::new);
```
스트림은 행위 단위로 내려쓰는 것이 가독성과 디버깅 차원에서 좋습니다.

Reply:
동료들이 보기에도 내리는게 좋을것 같네요 👍 
내려 쓰도록 연습해봐야겠네요 
감사합니다 😄

Reply:
동료들이 보기에도 내리는게 좋을것 같네요 👍
내려 쓰도록 연습해봐야겠네요
감사합니다